src/metabase/cmd.clj

```diff
@@ -168,53 +152,43 @@
 
   `--mode` can be one of `:update` or `:skip` (default). `--on-error` can be `:abort` or `:continue` (default)."
   [path & options]
-  (apply import* path false options))
+  (log/warn (u/colorize :red (trs "''load'' is deprecated and will be removed in a future release. Please migrate to ''import''.")))
+  (let [opts (merge {:mode     :skip
+                     :on-error :continue}
+                    (m/map-vals mbql.u/normalize-token (cmd-args->map options)))]
+    (call-enterprise 'metabase-enterprise.serialization.cmd/v1-load path opts)))
 
 (defn ^:command import
-  "Load serialized Metabase instance as created by the [[export]] command from directory `path`. Replaces the [[load]]
-  command. Options are the same as for `load`.
-
-  `--mode` can be one of `:update` or `:skip` (default). `--on-error` can be `:abort` or `:continue` (default)."
-  [path & options]
-  (apply import* path true options))
-
-(defn- export*
-  ([path v2?]
-   (export* path v2? "--state" :active))
-
-  ([path v2? & options]
-   (when-not v2?
-     (log/warn (u/colorize :red (trs "''dump'' is deprecated and will be removed in a future release. Please migrate to ''export''."))))
-   (let [options (cond-> options
-                   v2?
-                   (concat ["--v2" true]))
-         cmd     (resolve-enterprise-command 'metabase-enterprise.serialization.cmd/dump)]
-     (cmd path (cmd-args->map options)))))
+  "Load serialized Metabase instance as created by the [[export]] command from directory `path`."
+  [path]
+  (call-enterprise 'metabase-enterprise.serialization.cmd/v2-load path))
 
 (defn ^:command ^:deprecated dump
   "Deprecated: prefer [[export]] instead.
 
   Serialized metabase instance into directory `path`. `args` options may contain --state option with one of
   `active` (default), `all`. With `active` option, do not dump archived entities."
   [path & options]
-  (apply export* path false options))
+  (log/warn (u/colorize :red (trs "''dump'' is deprecated and will be removed in a future release. Please migrate to ''export''.")))
+  (let [options (merge {:state    :active
+                        :mode     :skip
+                        :on-error :continue}
+                       (cmd-args->map options))]
+    (call-enterprise 'metabase-enterprise.serialization.cmd/v1-dump path options)))
 
 (defn ^:command export
-  "Serialize a Metabase into directory `path`. Replaces the [[dump]] command. Options are the same as for `path`.
```

Comment:
I think `path` was supposed to be `dump` here, but it's probably not all of the options